src/OpenTelemetry.Exporter.ZPages/ZPagesExporterOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// <copyright file="ZPagesExporterOptions.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Exporter.ZPages
+{
+    /// <summary>
+    /// Options to run zpages exporter.
+    /// </summary>
+    public class ZPagesExporterOptions
+    {
+        /// <summary>
+        /// Gets or sets the port to listen to. Typically it ends with /statz like http://localhost:7284/statsz/.
```

Comment:
Let's start with spans (RPC) tracking. Call it rpcz
```suggestion
        /// Gets or sets the port to listen to. Typically it ends with /rpczz like http://localhost:7284/rpcz/.
```